src/youtube-player/youtube-player.spec.ts

```diff
@@ -0,0 +1,130 @@
+/// <reference types="youtube"/>
```

Comment:
I spent a considerable amount of time banging my head against a wall getting these files to recognize the @types/youtube that were downloaded with the yarn install. If there's another technique I'm all ears, but as near as I can tell, @types/youtube will be loaded if you call import on 'youtube', but since we only want the types, not the actual library itself, it wasn't able to load them...

Reply:
TypeScript is smart enough to trim `import` statements that are only used for type purposes, so importing the symbols directly should be fine. Does the code compile if you do that?

Reply:
(I think; it would be good to confirm)

Reply:
Yeah, if just type symbols are imported from the import, the import is never part of the generated JavaScript output. We generally do not use this construct for adding types.

Though it looks like in this case it's a bit different because the `@types/youtube` package seems to expose a `YT` namespace instead of an export for type symbols. Ideally the `tsconfig` for the youtube-player would just set the `types:` compiler option to contain the `youtube` types then.

This seems to be done already.. can you explain what breaks then? is it just bazel?

Reply:
Yeah, I was having quite a bit of typing issues, but I tried removing these and everything seems to build just fine... Maybe the problem was with the e2e app, and I'm not hitting the problem now since I removed that aspect from this PR. Either way, let's move forward without them and we can face the problem if it comes up again.